x-pack/legacy/plugins/siem/public/components/load_more_table/index.tsx

```diff
@@ -200,7 +204,7 @@ export class LoadMoreTable<T, U, V, W, X, Y, Z, AA, AB> extends React.PureCompon
         </EuiContextMenuItem>
       ));
     return (
-      <EuiPanel
+      <Wrapper
```

Comment:
This upset the browser, still trying to figure out how to fix this (Render `EuiPanel` from `Wrapper`).